Packs/ProofpointThreatResponse/ModelingRules/ProofpointThreatResponseModelingRules/ProofpointThreatResponseModelingRules.yml

```diff
@@ -0,0 +1,9 @@
+from_xdr_version: 3.4.0
+fromversion: 6.8.0
+name: Proofpoint Threat Response Modeling Rule
+id: proofpoint_threat_response_modeling_rule
+tags: proofpoint
+rules: ''
+schema: ''
+to_xdr_version: 3.4.9
+toversion: 6.9.9
```

Comment:
```suggestion
```
as Shahaf said last week, if the issues with the new modeling rules are not resolved until you are ready to merge, let's remove them and we'll add them later